internal/pkg/cli/pipeline_init.go

```diff
@@ -237,30 +238,48 @@ func (opts *InitPipelineOpts) createBuildspec() (string, error) {
 	return path, nil
 }
 
-func (opts *InitPipelineOpts) selectEnvironments(addMore bool) error {
+func (opts *InitPipelineOpts) selectEnvironments(addMore bool, firstPrompt bool) error {
```

Comment:
Instead of having a recursive function call, can we loop through the available environments and if it's the first index ask the `pipelineAddEnvPrompt`, otherwise ask the `pipelineAddMoreEnvPrompt`?

Reply:
Yeah. I was thinking about using the loop as well. However, the reason why I did not make that change is because actually using the recursive function does not affect the functionality at all and it might not be necessary to make that change.

Reply:
It's functionally equivalent, but we can get rid of the input bool and if/else by using a loop which would be much more readable.